### PR TITLE
Add Centos 8 support

### PR DESCRIPTION
**Pull request by Vancelot11**

### CODE REVIEW

**Comment by Vancelot11** [comment]
[osl-nfs](https://trello.com/c/8tEI04Ra/1564-osl-nfs)

File: Berksfile [hunk 1]
```diff
@@ -2,4 +2,8 @@ source 'https://supermarket.chef.io'
 
 cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall'
 
+# forked to add centos 8 support to upstream cookbook
+# https://github.com/atomic-penguin/cookbook-nfs/pull/114
+cookbook 'nfs', git: 'git@github.com:Vancelot11/cookbook-nfs', branch: 'vancelot/c8'
```
Comment (ramereth) [inline Berksfile]: Looks like this was merged and released: https://supermarket.chef.io/cookbooks/nfs#changelog Please remove this and update the metadata to ensure we grab the correct version.

**Comment by Vancelot11** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.0.4.  Have a nice day!

**Pull request merged**